pyproject.toml

```diff
@@ -259,6 +259,6 @@ networktest = "pytest --cov --cov-report=xml --cov-report=term --durations=0"
 coincident = { path = ".", editable = true }
 
 [tool.pixi.feature.docs.tasks]
-docs = "python -m sphinx --keep-going -T -b html -d docs/_build/doctrees -D language=en docs docs/_build/html"
-# NOTE: use https://jupyter-cache.readthedocs.io ?
-rebuild_docs = "rm -rf docs/_build ; python --keep-going -m sphinx -T -b html -d docs/_build/doctrees -D language=en docs docs/_build/html"
+# NOTE: || true to always update readthedocs even if there are errors
+docs = "python -m sphinx --keep-going -T -b html -d docs/_build/doctrees -D language=en docs docs/_build/html || true"
```

Comment:
This is in order to render all the examples that *do* work. Be aware that RTD workflow will always be green now even if notebooks fail to execute, so be sure to review preview links and logs